io.github.subhra74.Muon.json

```diff
@@ -0,0 +1,49 @@
+{
+	"app-id": "io.github.subhra74.Muon",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "23.08",
+	"sdk": "org.freedesktop.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.openjdk11"
+	],
+	"command": "snowflake",
+	"finish-args": [
+		"--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+		"--share=ipc",
+		"--socket=x11",
+		"--share=network",
+		"--filesystem=home"
```

Comment:
Why do you need home access?

Reply:
Because this app also supports SFTP, file transferring often provides cover home directory.

Reply:
Can't you make it specific a single directory like xdg-downloads or xdg-documents?

Reply:
This is not even very convenient for me personally, and some of the functions of the native application are cut off. Then it makes no sense for me to publish it in Flatpak.